### PR TITLE
[FE][Integration] Completing the undefined variable (id & router)

### PR DESCRIPTION
**Pull request by enggarlr**
### Description:
The last comitted games's index.js forgot to define "id" and "router".

### Type of Change:
- [X] Bug Fix
- [ ] New Feature
- [x] Refactor/Breaking Change

### CODE REVIEW

File: pages/games/index.js [hunk 1]
```diff
@@ -38,16 +30,15 @@ function Index() {
...
-                  description= {game.description}
-                  viewCount= {game.viewCount}
-                  playCount= {game.playCount}
-                  handlePlay= {handlePlay} 
-                />
-              </div>
+              <GameCard 
+                key={game.id}
+                id = {game.id}
+                thumbnails= {game.thumbnails}
```
Comment (grgsferry) [inline pages/games/index.js:34]: ini harusnya game.thumbnail mas

Comment (enggarlr) [reply pages/games/index.js:33]: oke siap mas, diperbaiki dulu

File: pages/games/index.js [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
+import { useRouter } from "next/router";
 import Navbar from '../../components/Navbar';
 import Footer from '../../components/Footer';
 import GameCard from '../../components/GameCard';
 import axios from 'axios';
 
 function Index() {
   const [games, setGame] = useState([]);
+  const router = useRouter();
+  const { id } = router.query;
```
Comment (grgsferry) [inline pages/games/index.js]: bagian ini masih kepake mass? kalo ga mungkin bisa ditakeout

Comment (enggarlr) [reply pages/games/index.js]: done mas

**Review by grgsferry — APPROVED** [review]

**Review by masegy — APPROVED** [review]

**Pull request merged**